lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -72,18 +73,47 @@ import {
 import { SilentCacheClient } from "./SilentCacheClient";
 import { AuthenticationResult } from "../response/AuthenticationResult";
 import { base64Decode } from "../encode/Base64Decode";
+import { version } from "../packageMetadata";
 
 const BrokerServerParamKeys = {
     BROKER_CLIENT_ID: "brk_client_id",
     BROKER_REDIRECT_URI: "brk_redirect_uri",
 };
 
+/**
+ * Sets MSAL and browser extension SKUs
+ * @param messageHandler {NativeMessageHandler}
+ */
+function setSKUs(messageHandler: NativeMessageHandler): string {
+    const groupSeparator = ",";
+    const valueSeparator = "|";
+    const skus = Array.from({ length: 4 }, () => valueSeparator);
+    // Report MSAL SKU
+    skus[0] = [BrowserConstants.MSAL_SKU, version].join(valueSeparator);
+
+    // Report extension SKU
+    const extensionName =
+        messageHandler.getExtensionId() === "ppnbnpeolgkicgegkbkbjmhlideopiji"
```

Comment:
Please use [constant](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/be05b0571b240c9053ea84dff5551bbce5d1f05b/lib/msal-browser/src/utils/BrowserConstants.ts#L46)